lib/generate_dart_api.dart

```diff
@@ -148,8 +152,11 @@ void _generateImportStub(
 /// generates a FileSummary for it.
 Future<FileSummary> _parseFile(String inputPath, GlobalConfig globalConfig,
     {bool ignoreFileErrors: false}) async {
-  var results = await Process.run(
```

Comment:
I thought I had deleted it! Now I've done it. Now I've done it, but I'm delaying the push until the review get finished to not invalidate other comments.